app/controllers/application_controller.rb

```diff
@@ -50,12 +50,6 @@ def display_search_box_in_header?
 
   helper_method :display_search_box_in_header?
 
-  def display_menu_button_in_header?
-    true
-  end
-
-  helper_method :display_menu_button_in_header?
-
```

Comment:
Did you search if the engines uses this helper method? (I am just asking because I faced this problem before on removing a code on the app controller that the engines depends). Perhaps the engines don't use it, but it is good to check.